Tuist/Package.swift

```diff
@@ -30,11 +30,15 @@ let package = Package(
     .package(
       url: "https://github.com/CSolanaM/SkeletonUI.git",
       revision: "f025e9ba5d051374d7344efa8859df5b9399a181" // 2.0.2
+    ),
+    .package(
+      url: "https://github.com/kakao/kakao-ios-sdk.git",
+      branch: "master"
```

Comment:
```suggestion
      revision: "ab4309c1950550add307046ad1e08024c7514603" // 2.23.0
```

다른 서드파티 라이브러리들이랑 동일하게 **revision**으로 맞추는게 더 좋을것 같아요!